python-test2.py

```diff
@@ -1,61 +1,83 @@
 import random
 
 class InventoryItem:
```

Comment:
-    def __init__(self, name, description, price):
-        self.name = name
-        self.description = description
-        self.price = price
-
-    def __str__(self):
-        return f"{self.name}: {self.description} ({self.price})"
-
-    def __repr__(self):
-        return f"{self.name}: {self.description} ({self.price})"
-
-
-def generate_random_inventory_item():
-    names = [
-        "Sword",
-        "Shield",
-        "Helmet",
-        "Armor",
-        "Potion",
-    ]
-    descriptions = [
-        "A sharp steel sword.",
-        "A sturdy wooden shield.",
-        "A polished iron helmet.",
-        "A set of chainmail armor.",
-        "A healing potion.",
-    ]
-    price = random.randint(10, 100)
-    return InventoryItem(
-        random.choice(names),
-        random.choice(descriptions),
-        price,
-    )
-
-
-def main():
-    item = generate_random_inventory_item()
-    print(item)
-
-
-if __name__ == "__main__":
-    main()
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-



































-
-
-
-
-
-
-
-


-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
















 \

 \

 \

 \





































 \

 \

 \

 \






































































































































































 \

 \

 \

 \





















 \

 \

 \







 \

 \

 \

 \

 \

 \

 \



















 \

 \

 \

 package































































































































-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-


-
-
-
-
-
-
-
-
-
-


-
-
-
-


-
-
-
-
-
-
















































































































































































 \

 \

 \

 \

































































































































































































































































































































































































 \n
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
- #
-
-
-
-
-
-
-
-
-
-
-
-
- #
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-






















































































 \











































 \

 \









































































































































































 \

 \

 \

 \

















































































 \

 \

 \















































































































 \





























 \










 \

 #
-

 #
-

 \

 #
-

 \



 \

 \

 \











 \

 #
-

 \

 \

 #
-

 #
-

 \

 \

 \

 \

 \

 \

 #
-

 \

 #
-

 \

 #
-










































































 \

 #





















 \

 #
-

 \

 \

 #
-
-
-

 #
-

 \

 #
-

 \

 \









 \

 #
-

 \

 #
-
-

 \

 #
-

 \

 \

 \
























 \

 \

 \

 \

 \

 \












 \

 #
-
-

 \

 \

 \

 \

 \

 \

 \

 \

 #
-





















 \

 \

 \

 \

 #
-
-
-











 \

 \

 \






































 \

 #

 #
-
-

 #

 \

 #
-

 \

 \

 #
-





 \

 #
-

 \

 \

 \

 \

 \

 \




 \

 \

 \

 \




















 \

 #

 \

 \

 \

 \

 #
-

 \

 #

 \

 #

 \

 #

 \

 \

 #
-

 \

 \

 \

 \

 #
-

 \

 \

 #
-

 #
-
-

 #
-

 \

 #
-

 \

 \

 #
-

 \

 #
-

 \

 \

 #

 \

 \

 #
-

 \

 \

 \

 \

 \

 \

 \




 \

 \

 \

 #
-

 \

 \

 #
-

 \




 \

 \




























 \
























 \

 \

 \



































































































































































 \

 #
-
-

 \

 #
-




 \

 #
-

 \

 \

 \

 \

 \









































 \

 \

 \

 \






















 \

























 \

 #
-
-

 \n


















 \

 \






 \

 \

 #
-








 \

 \














 \

 \

 \

 \

 \

 \

 #
-



 \

 \

 #
-
-
-

 \

 \












 \

 \

 #
-





















 \












































 \

 \










 \












































 \

 \






 \

 \

 \






























 \


















 \

 \

 \

 \

 \

 \

 #
-
-
-
-
-

 \

 \

 \

 \






 \

 \

 \

 \





























 \

 \

 \

 \


































 \

 #
-
-
-
-
-

 \

 #
-
-
-
-
-
-
-













 \

 #
-
-
-
-

 \n

 \

 \






















































 \

 \






 \

 \

 \

 \

 \

 \





















 \

 \

 \

 \

 \

























 \

 \

 \

 \

 \

 \

 \

 \

 \















































































































 \

 \













































 \

 \

 \

 \


















































































 \

 #
-

































 \





 \

 \

 \

 \

 #
-
-
-
-

 \

 \

 \

 #
-

 \

 \

 \

 \

 \











 \

 \

 \

 \

 \

 \

 \

 \

 #
-
-
-
-
-

 \

 \












 \

 #
-
-
-
-
-

 \

 \

 \



















































































































 \

 \

 \

 # # \

 # #
-
-
-
-
-
-

 \

 # #
-
-

 \

 \

 #
-





















 \

 \

 \

 \

 #
-
-
-

 \

 #
-



 \

 \

 \
















 \

 \

 #
-
-






 \

 \

 \

 \

 \

 \

 \

 \




 \

 #
-

 \

 #
-


 \

 \

 \

 \




















 \

 \


























 \

 \




 \

 \

 #
-
-

 \















 \

 \

 \

 \














































 \

 \

 # # \

 # #
-
-

 \

 \

 \

 \

 \

 \

 # #
-
-








 \

 \

 \

 \

 \

 \

 \





 \

 \

 # #
-
-
-

 \

 \

 \

 \




 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 # #
-
-
-
-
-

 \

 # #
-
-
-
-

 # \

 \

 \

 # #
-
-

 \

 \

 \

 \

 \

 # #
-
-
-
 \

 \

 # #
-
-
 \

 \

 \

 \

 \

 \

 \

 \

 # #
-
-
 \

 \

 \

 # #
-
-

 \

 # #
-
-
-


 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # #
-


 \

 \

 \

 \

 \

































































 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \



































 \

 \

























































































































 \

 \

 \

 \















 \

 \

 \

 \

 # # \

 \

 # #
-

 \

 # #
-
-
-

 # #

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

















 \

 \

 \

 # \

 \

 \

 \

 \

 \

 \


















 \

 \

























 \

 \

 # # \

 \

 \

 \

 \

 \

 \


































 \

 \

 \
























 \n
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \








 \

 \

 \

 \

 \

 # # \

 \














 \

 \

 \

 \




















































































 \

 # # \

 \

 \

 \

 \

 \

 \


























 \

 \





















 \

 \

 \

 \

 \

 \

 \

 \







 \

 \

 \

 \
















 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 \

 \

 \














 \

 \

 \

 \

 \

















 \

 \











 \

 \

 \

 \




















































 \

 \

 \

 \




 \

 \







 \

 \

 \

 \

 \














 \

 \















 \

 \



















































































 \

 \



 \

 \

 \

 \




















 \

 \

 \

 \







 \

 \
































 \

 \




 \

 \

 \





 \

 \

 \

 \

 \




 \




























 \

 \

 \

 \





 \

 \









































































 \

 \

 \

 \

 \

 \






















 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \












 \

 \






 \

 \

 \

 \

 \

 \

 \

 \





















 \

 \

 \

 \

 \

 # # \

 \

















 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 \

 \

 \

 # # \

 # #
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \













 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 # # \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \










 \
























 \










 \

 \

 \

 \







 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 # # \

 # #
-
 \

 \

 \

 \

 # #
 \

 \

 # # \

 # #
-

 \

 # #
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 # # \

 # #
-
 \

 \

 # # \

 \

 \

 # #
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 \

 \

 # # \

 # #
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \









 \

 # # \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \



























 \n
 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 \

 \










 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 \

 \

 # # \

 # #
 \

 \

 # #
 \

 \

 # #
 \

 \

 # # \

 # #
 \

 \

 # #
 \

 # #
 \

 \

 \

 # #
 \

 # #
 \

 \

 \

 \

 \

 \

 # #
 \

 # #
 \

 # #
 \

 # #
 \

 \

 \

 # #
 \

 \

 \

 \

 \

 # #
 \

 # #
 \

 # #
 \

 # #
 \

 \

 # #
 \

 \

 \

 # #
 \

 \

 \

 \

 \

 # #
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

























 \n
 \

 \

 \

 \

 \

 \















 \n
 \





 \

 \

 \

 \

 \

 \

 \

 \

 \



 \

 \











 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \









 \

 \

 \

 \









 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \





 \

















 \n
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 \

 \

 \

 # # \

 \

 \

 # #
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \


















 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \





















 \

 \

 \

 \

 \

 \














 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 # # \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # \

 \

 \

 \

 # \

 \

 # \

 \n
 \

 # \

 \n
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # \

 \

 \

 \

 \

 \

 \

 \

 # \

 \

 # \

 \

 \

 # # \

 # #
 \

 \

 # \

 \n
 \

 \

 # #
 \

 \

 # #
 \

 \

 # \

 # \

 # \

 # \

 # \

 # \

 # \

 \

 # \

 \

 \

 \

 # \

 # \

 # #
 \

 \

 # #
 \

 \

 # #
 \

 # #
 \

 # \

 \

 # \

 # \

 # \

 # \

 \

 # \

 \

 # #
 \

 # #
 \

 # #
 \

 # #
 \

 # #
 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # # \

 \

 # \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 \

 # \

 \

 # \

 \

 # \

 # \

 # \

 # \

 \n
 \

 \

 # \

 \

 # \end #
 \

 \n
 \

 \

 \

 \

 \

 # \

 # #
 \

 \

 # \

 # \end #
 \

 # \